Gemfile

```diff
@@ -96,6 +96,8 @@ gem "mimemagic"
 
 # Utils
 gem "mustermann" # Used by the autumn moon bot framework for pattern matching
+gem "exception_notification"
```

Comment:
the website for this gem does a redirect and you can't hit the back button because it just does the redirect again.

i dont trust it.

Reply:
welp